covertest.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
```

Comment:
It's from notary.  String mangling and dealing with sets was just easier in Python.